Problem1.rb

```diff
@@ -0,0 +1,22 @@
+# sum of evem numbers from integer array
+
+def SumOfEven numbers =[]
+sum=0
+
+    if numbers.nil?
+        raise ArgumentError, 'Argument is null' 
+    end  
+   
+    numbers.each do|number|
+        if !number.is_a? Integer
+           raise ArgumentError, 'Argument is not numeric' 

```

Comment:
The problem statement does not mention anything else then numbers in the array, I don't this brings any value, in fact in makes the code harder to read.

Using negation here make the code harder to reason about, there are better ways to do this in ruby, can you think of one?

Reply:
The above question is still valid, please let me know what will be the alternative to a negating if in ruby?

Reply:
if not  or unless statements